MiniProject1.py

```diff
@@ -0,0 +1,96 @@
+# Kimber's file
+from pattern.en import *
+rhymeWords=dict() #dictionary of words that rhyme
+generalWords=dict() #markov chain dictionary
+#fo = open("ShakespeareSonnets.txt", "r")
+with open('ShakespeareSonnets.txt', 'r') as f:
+	read_data = f.read()
+f.closed
```

Comment:
What is this line for? Are you trying to ensure that the file is closed? There are interesting error throwing ways of doing this, but you probably don't need this line, since the 'with' keyword and the open method together handle file closing.